specification/Metadata/0.0.0/README.md

```diff
@@ -1209,17 +1267,37 @@ Furthermore, depending on the implementation, the method of specifying a texture
-* **geometry** - Any geometric entity such as a vertex, texel, or larger structures like meshes or patches of texels.
-* **metadata** - Application-specific properties associated with geometry.
-* **class** - A high-level description of one or more associated pieces of metadata (called properties).
-* **property** - A description of a single piece of metadata. This includes information such as a datatype.
-* **instance** - A concrete representation of a class consisting of a value for every property.
-* **instance table** - A mapping of instance IDs to metadata for each instance. The values are stored in parallel property arrays.
 * **buffer** - a contiguous sequence of bytes used for storing binary data. This is equivalent to the [glTF `buffer` concept](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#buffers-and-buffer-views).
-* **buffer view** or `bufferView` - A subsequence of a buffer that represents a single array. A buffer view is completely contained within a buffer. This is equivalent to the [glTF `bufferView` concept](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#buffers-and-buffer-views), though this specification uses bufferViews for a broader set of data types.
-* **property array** - A single array that stores values for one property.  
+* **buffer view** or `bufferView` - A subsequence of a buffer that represents a single array. A buffer view is completely contained within a buffer. This is equivalent to the [glTF `bufferView` concept](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#buffers-and-buffer-views), though applied to a broader set of data types.
+* **class** - A high-level description of one or more associated pieces of metadata (called properties).
+* **component** (of an element) - For elements of type `ARRAY`, the values contained within are called components. In other words, a property array contains elements which contain components.
 * **element** (of a property array) - a single entry of a property array. This stores the value of one property for a single instance.
-* **component** (of an element) - For elements of type `ARRAY`, the values contained within are called components. In other words, a property array contains elements which contain components. 
-* **metadata texture** - A texture that represents a property directly, no instance table needed
-* **encoding** - A method of storing metadata on disk in a format such as JSON, binary, or texture encodings.
\ No newline at end of file
+* **encoding** - A concrete method of storing metadata in a format such as JSON, binary, or textures.
+* **instance** - A concrete encoding of a class consisting of a value for every property of the class.
+* **instance table** - A table of metadata values indexed by instance IDs. The columns of the table are a set of parallel property arrays.
+* **metadata** - Additional application-specific properties in a structured format.
```

Comment:
Another question about consistent terminology: is metadata "scoped to", "assigned to", "attached to", "associated with", or "added to" data?